.github/no-response.yml

```diff
@@ -0,0 +1,14 @@
+# Configuration for probot-no-response - https://github.com/probot/no-response
+
+# Number of days of inactivity before an Issue is closed for lack of response
+daysUntilClose: 30
+
+# Label requiring a response
+responseRequiredLabel: need-more-info
+
+# Comment to post when closing an Issue for lack of response. Set to `false` to disable
+closeComment: >
+  This issue has been automatically closed because there has been no response
+  to our request for more information from the original author. With only the
```

Comment:
Could this say "original author or interested parties"?

Reply:
Not in this case: the _No Response_ bot is used to require answers from the original author when issue is incomplete, whereas the _Stale_ bot accepts updates from anyone to keep the issue alive.